src/auro-select.js

```diff
@@ -237,6 +250,19 @@ class AuroSelect extends LitElement {
     }
   }
 
+  /**
+   * Determines the element error state based on the `required` attribute and input value.
+   * @private
+   * @returns {void}
+   */
+  generateOptionsArray() {
+    if (this.menu) {
+      this.options = [...this.menu.querySelectorAll('auro-menuoption')];
+    } else {
+      this.options = [];
+    }
+  }
```

Comment:
@jordanjones243 @jason-capsule42 was this update tested with a non-static list of options? Will this update if the list of options is updated from JavaScript dynamically in the page?